packages/settings-library/src/settings_library/prometheus.py

```diff
@@ -20,3 +20,7 @@ def base_url(self) -> str:
             port=f"{self.PROMETHEUS_PORT}",
             path=f"/api/{self.PROMETHEUS_VTAG}/query",
         )
+
+    @cached_property
+    def origin(self) -> str:
```

Comment:
What is the use of that property? what is meant by origin?

Reply:
explanation below